### ISSUE TITLE
Implement new `combine` and `separate` actions

### ISSUE THREAD

**Issue by pospi** [holo-rea/holo-rea#192]
These provide a way for `EconomicEvents` to modify the `containedIn` / `contains` properties of a resource.

- [ ] The ability to edit these fields directly should also be removed.

See:
- https://lab.allmende.io/valueflows/vf-schemas/vf-graphql/-/issues/95
- https://lab.allmende.io/valueflows/valueflows/-/issues/637
-  https://valueflo.ws/examples/ex-production.html#pack-unpack

**Comment by fosterlynn** [comment]
Update from the VF side:  We're working through pack/unpack in detail in a use case in Reflow.  I have a gut feel we could improve the way we envision it playing out: instead of following the container resource out of the process that packs it, create a new package resource, which embodies the container (if used) and all the packed resources.   Anyway, best to let this issue sit until we get through this use case.  But, if you find you need it for something, let us know and we'll expedite.

**Comment by Connoropolous** [comment]
@fosterlynn 
is this no longer pack/unpack? 

Is it combine/separate instead? 
https://www.valueflo.ws/concepts/actions/#action-behaviors

**Comment by fosterlynn** [comment]
>Is it combine/separate instead?

Yes, correct.  Sorry if this didn't get communicated all the way through.

**Comment by Connoropolous** [comment]
ok cool. renamed the issue
